static-build/test/static-build/exports.test.lua

```diff
@@ -47,7 +47,6 @@ local check_symbols = {
 
     'uuid_nil',
     'tt_uuid_create',
-    'tt_uuid_str',
```

Comment:
Nit: It seems logical to add `tt_uuid_to_string` instead.

Reply:
I don't object.